app/controllers/items_controller.rb

```diff
@@ -5,7 +5,9 @@ def index
     @items = Item.unscoped.all.order(id: :asc)
```

Comment:
unscoped is used several times so use a more specific method or flag to retrieve active items.